src/api/[...auth].ts

```diff
@@ -2,4 +2,10 @@ import { AstroAuth } from '../../server'
 
 export const prerender = false
 
-export const { GET, POST } = AstroAuth()
+export async function GET(context) {
+    return await AstroAuth().GET(context)
+}
+
+export async function POST(context) {
+    return await AstroAuth().POST(context)
+}
```

Comment:
Why are you splitting this up and calling it twice? Unless I'm missing something, the end result is the same, except the `options` object is now created in two different scopes.